### ISSUE TITLE
Galleria: Next & Previous Buttons role

### ISSUE THREAD

**Issue by e-matthaey** [primefaces/primeng#16446]
### Describe the bug

To improve accessibility of the galleria component, the following points should be adjusted:
- next & previous buttons are not allowed to have a role="navigation"
- both buttons need an aria-label

![primeng-galleria](https://github.com/user-attachments/assets/2fa8def1-4aa5-4fd5-b43a-832ec1d9583c)

### Environment

Angular > 18
PrimeNG > 17

### Angular version

18

### PrimeNG version

17.18.10

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

22
